jcl/src/java.base/share/classes/java/lang/invoke/MethodHandles.java

```diff
@@ -681,6 +732,25 @@ public MethodHandle findVirtual(Class<?> clazz, String methodName, MethodType ty
 			handle = SecurityFrameInjector.wrapHandleWithInjectedSecurityFrameIfRequired(this, handle);
 			return handle;
 		}
+
+		/*[IF Sidecar19-SE]*/
+		/**
+		 * Check if targetClass is in a package visible from the accessModule
+		 * @param accessModule module of the referring class
+		 * @param targetClass Class which the referring class is accessing
+		 * @throws IllegalAccessException if the targetClass is not visible
+		 */
+		static void checkClassModuleVisibility(Module accessModule, Class<?> targetClass) throws IllegalAccessError, IllegalAccessException {
```

Comment:
This probably also needs to take the `UNCONDITIONAL` access bit into account (ie: public lookup)